packages/nimble-components/src/rich-text-viewer/index.ts

```diff
@@ -11,7 +18,99 @@ declare global {
 /**
  * A nimble styled rich text viewer
  */
-export class RichTextViewer extends FoundationElement {}
+export class RichTextViewer extends FoundationElement {
+    /**
+     * Whether to grow the height of the component to fit the content.
+     *
+     * @public
+     * @remarks
+     * HTML Attribute: fit-to-content
+     */
+    @attr({ attribute: 'fit-to-content', mode: 'boolean' })
+    public fitToContent = false;
+
```

Comment:
Seeing it in practice I actually don't think the viewer should have the fit-to-content option enabled.

The viewer is most analogous to a `<p></p>` tag with a bunch of stuff inside of it. So making a block level element (with layout `flex`) makes sense. And a `<p>` tag will by default grow to fill the height of the content, you don't need to set some property on it. Instead if you want it to be a fixed height you would set a height on it explicitly.

The editor will be most analogous to a `<textarea>` in which case it is an inline element, not a block element so has a width and height by default. That one will need the `fit-to-content` attribute so that it grows to the size of the content entered which is a new behavior for a textarea type control.

If that approach makes sense make sure to update the spec to remove the fitToContent property from the viewer. Can include that change in this PR.